src/ProjectTemplates/Web.ProjectTemplates/RazorPagesWeb-CSharp.csproj.in

```diff
@@ -32,5 +32,10 @@
     <PackageReference Include="Microsoft.Identity.Web.DownstreamApi" Version="${MicrosoftIdentityWebDownstreamApiVersion}" Condition=" '$(IndividualB2CAuth)' == 'True' OR '$(OrganizationalAuth)' == 'True'" />
   </ItemGroup>
 
+  <!-- Temporarily ignore RID-related warnings. -->
+  <ItemGroup>
+    <RuntimeHostConfigurationOption Include="System.Runtime.Loader.UseRidGraph" Value="true" />
+  </ItemGroup>
```

Comment:
This will actually show up in people's real apps when they create them, won't it? Is that OK?

Reply:
Yes, it will. I gave this a try largely to see if this is the only remaining issue with this SDK update or if we might have to onion peel on another item.

As to whether or not it is OK, it appears that the right thing to do is resolve the underlying issue is resolve https://github.com/ericsink/SQLitePCL.raw/issues/543.

Given that we've branched for preview7 already, I was assuming that it was OK to take this in as we have some runaway to land a fix in preview7.

@bricelam Thoughts on feasibility of getting a fix for the underlying issue?

Reply:
It looks like we'll also need to fix templates in the spa-templates repo. :/

I'm fine to pursue this assuming we think we can land a permanent fix for preview7 or there isn't a way to address this that is more hidden from the user.